README.md

```diff
@@ -60,31 +97,42 @@ signed_prekey = state.SignedPreKeyRecord(
 store.save_signed_pre_key(signed_pre_key_id, signed_prekey)
 ```
 
-## Sending a message to a new participant
+### Sending a message to a new participant
+
+With a client initialized, you can create a session and send messages.
+
+To create a session, you must fetch a prekey bundle for the recipient from the server.
+Here the prekey bundle is `recipient_bundle` for participant `recipient_address`.
 
 ```py
 from signal_protocol import session, session_cipher
 
-# To create a session, you must fetch a prekey bundle for the recipient from the server
-# Here the prekey bundle is `recipient_bundle` for participant `recipient_address`
 session.process_prekey_bundle(
     recipient_address,
     store,
     recipient_bundle,
 )
+```
 
-# Once the prekey bundle is processed (storing data from the recipient in your local
-# protocol store), you can encrypt messages
+Once the prekey bundle is processed (storing data from the recipient in your local
+protocol store), you can encrypt messages:
+
+```py
 ciphertext = session_cipher.message_encrypt(store, recipient_address, "hello")
```

Comment:
the conflict from the main branch is just because i changed the API here to take the message in bytes, so it's now: 

`ciphertext = session_cipher.message_encrypt(store, recipient_address, b"hello")	`